data/the emerald sword.txt

```diff
@@ -0,0 +1,850 @@
+	to offer
+		has "paid mammon"
+		credits >= 10000000
+
+	on offer
+		payment -10000000
+		conversation     
+			`You receive a message informing you of a very large transaction: 10 million credits. Mammon's crew have withdrawn the money you paid for the information about the box.`
+				decline
+
+mission "Sheragi Archaeology: The Box 5"
+	landing
+	name "The Fight for the Artifact"
+	description "Destroy the mercenary fleet, then land on <planet>."
+	source "Watcher"
+	
+	to offer
+		has "Sheragi Archaeology: The Box 4: done"
+	
+	on offer
+		conversation
+			`You arrive on the uninhabited moon of Watcher and scout the area Mammon pointed out. It's relatively easy to find some smoke rising from a campfire in a clearing, but as you approach to investigate, you're surprised to find some large ships taking off to greet you, ditching the green tarps used to hide among the trees. Looks like they really don't want you here.`
+				launch
+	
+	npc disable
+		government "Bounty Hunter"
+		personality unconstrained launching nemesis heroic plunders harvests target marked
+			confusion 20
+		system "Algieba"
+		fleet "Marauder fleet X"
+		dialog
+			`You managed to destroy the entire mercenary fleet. You should have no problem landing on their camp now and recovering the cube.`
+	
+	on visit
+		conversation
+			`You attempt to evade the mercenary ships and make a quick assault on their encampment, but the ships follow you through the atmosphere, and ground personnel start shooting at you. Bombing it on a pass means you risk destroying the cube. You have no choice but to go back to space and eliminate the mercenary fleet.`
+				launch
+	
+mission "Sheragi Archaeology: The Box 6a"
+	landing
+	invisible
+	source "Watcher"
+	destination "Vinci"
+	
+	to offer
+		has "Sheragi Archaeology: The Box 5: done"
+	
+	on offer
+		conversation
+			`You come in for a landing, and the few mercenaries that stayed on the ground march to your ship carrying a makeshift white flag. Dozens of men and women, tough looking and in uniform, unarmed, remaining peaceful and organized. These weren't ordinary criminals, it seems.`
+			`	"We know when we've lost," one of them yells to your ship on a megaphone. "What is it that you want?"`
+			choice
+				`	(Shoot them.)`
+					goto shoot
+				`	(Talk to them.)`
+			
+			`	You land and cautiously open the hatch to talk with them.`
+			`	"Who do you work for? Where is the cube you stole?" you ask.`
+			`	"So that cube was yours?" One woman up front says. "I knew that mission was sketchy... We're mercenaries. We were employed by some men in suits who paid us really well to steal it, yet when we returned with it, they just scanned it and told us to dispose of it. Said that a digital copy should be enough, and the trouble should be enough retribution. We don't know who they were, we don't ask questions. That might be about to change now.`
+			`	"Anyway, we'll give you the cube on good terms if you spare us."`
+			choice
+				`	(Spare them.)`
```

Comment:
This whole bit here would be interesting if it either gave you or cost you some amount of Karma. Also, if you spare them, a positive relationship with a mercenary faction that could show up in future missions might be very interesting.

Reply:
A previous version of that quest had some karma cost involved, but people were against it since karma is only used for the main questline, and this is technically a sidequest. karma also only affects one mission in the end. maybe if the system was more fleshed out, it would be good, but people would need to agree on some stuff. It's a great discussion to have.